packages/framework/src/Controller/Admin/ProductController.php

```diff
@@ -372,4 +408,18 @@ public function visibilityAction($productId)
             'domains' => $this->domain->getAll(),
         ]);
     }
+
+    /**
+     * @return bool
+     */
+    protected function productCanBeCreated()
```

Comment:
please consider return type `protected function productCanBeCreated(): bool`

Reply:
that class does not use strict types so I will not use it here